### ISSUE TITLE
How to train the kp_detector model only?

### ISSUE THREAD

**Issue by GesilaA** [AliaksandrSiarohin/first-order-model#454]
The architecture of kp_detector defined in 'modules/keypoint_detector.py', and applied in 'GeneratorFullModel'. Loss function contains 'perceptual', 'generator_gan', 'feature_matching', 'equivariance_value', and 'equivariance_jacobian'. Which loss are designed for the kp_detector training？How to separate the keypoint model from 'GeneratorFullModel'? I'm looking forward for your replies.

**Comment by AliaksandrSiarohin** [comment]
You can not separate kpdetector from the rest. The most important aspect is the fact that network is able to reconstruct the driving image from the driving keypoints, so perceptual loss is the most important. However you can not separate it from generator training.

**Comment by GesilaA** [comment]
@AliaksandrSiarohin Thank you for your rapid reply! Is 'perceptual' loss used to learn the unsupervised keypoints, 'equivariance_value' and 'equivariance_jacobian' losses are designed for keypoints stabilizing? Or all of the losses would affect the keypoints learning?

**Comment by AliaksandrSiarohin** [comment]
All of them affect keypoint training.
